src/stream-ciphers.rst

```diff
@@ -433,49 +433,48 @@ Bob decrypts :math:`C^{\prime}`, and gets the three plaintext blocks
                 & = P_1 \xor IV
    \end{aligned}
 
-:math:`R` is some random block. Its value doesn't matter.
+:math:`R` is a random block. Its value does not matter.
 
-Under the chosen-ciphertext attack assumption, Mallory recovers that
-decryption. She is only interested in the first block
+Mallory recovers the decryption under the chosen-ciphertext attack assumption.
+She is only interested in the first block
 (:math:`P^{\prime}_1 =
 P_1`) and the third block (:math:`P^{\prime}_3 = P_1 \xor IV`). By
-XORing those two together, she finds
-:math:`(P_1 \xor IV) \xor P_1 = IV`. But, the IV is the key, so Mallory
-successfully recovered the key by modifying a single message.
-
-Lesson learned: don't use the key as an IV. Part of the fallacy in the
-introduction is that it assumed secret data could be used for the IV,
-because it only had to be unpredictable. That's not true: “secret” is
-just a different requirement from “not secret”, not necessarily a
-*stronger* one. It is not generally okay to use secret information where
-it isn't required, precisely because if it's not supposed to be secret,
+XORing the two together, she finds
+:math:`(P_1 \xor IV) \xor P_1 = IV`. IV is the key, so Mallory
+successfully recovers the key by modifying a single message.
+
+Lesson learned: do not use the key as an IV. Part of the fallacy in the
+introduction is assuming secret data can be in the IV given
+it only has to be unpredictable. This is not true: “secret” is
+just a different requirement from “not secret”. It does not mean a
```

Comment:
"It does not mean a stronger one" makes no sense to me.

Reply:
This is a seoncd "Lesson learned" for attack on CBC mode.

I think this whole paragraph can be reworked a bit. Maybe this can be shortened and just reiterate the importance of picking random, unpredictable IVs.

And add in a general good guide on how to generate random, unpredictable IVs? (I don't know giving a guide is a thing for this book though)